### PR TITLE
11 organize dirs

### PR DESCRIPTION
**Pull request by deetz99**
issues: #11, #24

- Refactor to use assets from nuxt layer (tailwind theme, logo, favicon, dependencies) 
- reconfigure eslint with tailwind plugin for intellisense
- add product directories in /public, /content and /components/content
- homepage/default layouts
- initial i18n config (only en-CA for now)

Components:
- Header/Footer
- Side navigation using content folders
- LocaleSelect menu
- ColorModeSelect button
- Overwrite prose components to match theme

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://apigw-dev--pr-41-v310jblj.web.app

**Review by thorwolpert — COMMENTED** [review]
Good submission, just a couple of comments to add.

File: web/site/app.config.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      rounded: '',
+      font: '',
+      ring: 'focus-visible:ring-inset focus-visible:ring-2 focus-visible:ring-primary-500 dark:focus-visible:ring-white',
+      active: 'text-primary-500 dark:text-[#7BB5EF] border-[#94A3B8] dark:border-[#7BB5EF] font-semibold',
+      inactive: 'border-transparent hover:border-gray-900 dark:hover:border-white text-gray-700 hover:text-gray-900 dark:text-gray-400 dark:hover:text-white'
+    }
+  }
+})
+
+// declare module '@nuxt/schema' {
```
[the remark this answers is not in the document]
Comment (deetz99) [reply web/site/app.config.ts:45]: This is for typing the output of app config if using it as a layer/module, commented out as its not needed now but I can remove it.

File: web/site/components/Sbc/SideNavigation.vue [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                :links="item.children"
+              />
+            </template>
+          </UAccordion>
+        </div>
+      </nav>
+    </aside>
+  </div>
+</template>
+<i18n lang="json">
```
[the remark this answers is not in the document]
Comment (deetz99) [reply web/site/components/Sbc/SideNavigation.vue:77]: Drop in some comments?

File: web/site/components/content/ButtonDownloadSpec.vue [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
-    Download the specification
-  </a>
+  <UButton
+    label="Download the specification"
+    :to="href"
+    size="xl"
+    class="tracking-wide no-underline"
+  />
 </template>
+<i18n lang="json">
```
[the remark this answers is not in the document]
Comment (deetz99) [reply web/site/components/content/ButtonDownloadSpec.vue]: Translations: [0fabb57](https://github.com/bcgov/sbc-apigw/pull/41/commits/0fabb57a5276f8ece27363176a9d622c08c5c632)

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://apigw-dev--pr-41-v310jblj.web.app

**Review by thorwolpert — APPROVED** [review]

**Pull request merged**